web/html/xui/form/tls_settings.html

```diff
@@ -171,7 +171,7 @@
                     <template slot="title">
                         <span>{{ i18n "reset" }}</span>
                     </template>
-                    Short ID
+                    Short IDs
```

Comment:
We have just one short id not like alireza 3-4 ids 
So it's not correct
Undo your change